examples/setup_model_structure.py

```diff
@@ -6,7 +6,7 @@
 
 from oemoflex.model.datapackage import EnergyDataPackage
 
-# select scenario
+# select scenario key
```

Comment:
These are examples, not scenarios (and thus no scenario keys). Should rename the examples on a a separate branch. Here, just leave out the "key" again.

Reply:
Done in [0da658b](https://github.com/rl-institut/oemof-B3/pull/147/commits/0da658bd4567e29e1929d6f1cb95a9459975a051)